tutorial/westeros/westeros_addon_technologies.ipynb

```diff
@@ -0,0 +1,607 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model = 'Westeros Electrified'\n",
+    "base = message_ix.Scenario(mp, model=model, scenario='baseline')\n",
+    "scen = base.clone(model, 'addon_technology', 'illustration of addon formulation', keep_solution=False)\n",
+    "scen.check_out()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Retrieve parameters to perform subsequent addition of parameters"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "year_df = scen.vintage_and_active_years()\n",
+    "vintage_years, act_years = year_df['year_vtg'], year_df['year_act']\n",
+    "model_horizon = scen.set('year')\n",
+    "country = 'Westeros'\n",
+    "gdp_profile = pd.Series([1., 1.5, 1.9], index=pd.Index([700, 710, 720], name='Time'))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Define helper dataframes used for subsequent operations"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "base_input = {\n",
+    "    'node_loc': country,\n",
+    "    'year_vtg': vintage_years\n",
+    "    ,\n",
+    "    'year_act': act_years,\n",
+    "    'mode': 'standard',\n",
+    "    'node_origin': country,\n",
+    "    'commodity': 'electricity',\n",
+    "    'time': 'year',\n",
+    "    'time_origin': 'year',\n",
+    "}\n",
+    "\n",
+    "base_output = {\n",
+    "    'node_loc': country,\n",
+    "    'year_vtg': vintage_years,\n",
+    "    'year_act': act_years,\n",
+    "    'mode': 'standard',\n",
+    "    'node_dest': country,\n",
+    "    'time': 'year',\n",
+    "    'time_dest': 'year', \n",
+    "    'unit': '%',\n",
+    "}\n",
+    "base_capacity_factor = {\n",
+    "    'node_loc': country,\n",
+    "    'year_vtg': vintage_years,\n",
+    "    'year_act': act_years,\n",
+    "    'time': 'year',\n",
+    "    'unit': '%',\n",
+    "}\n",
+    "base_technical_lifetime = {\n",
+    "    'node_loc': country,\n",
+    "    'year_vtg': model_horizon,\n",
+    "    'unit': 'y',\n",
+    "}\n",
+    "base_inv_cost = {\n",
+    "    'node_loc': country,\n",
+    "    'year_vtg': model_horizon,\n",
+    "    'unit': 'USD/GWa',\n",
+    "}\n",
+    "base_fix_cost = {\n",
+    "    'node_loc': country,\n",
+    "    'year_vtg': vintage_years,\n",
+    "    'year_act': act_years,\n",
+    "    'unit': 'USD/GWa',\n",
+    "}\n",
+    "base_var_cost = {\n",
+    "    'node_loc': country,\n",
+    "    'year_vtg': vintage_years,\n",
+    "    'year_act': act_years,\n",
+    "    'mode': 'standard',\n",
+    "    'time': 'year',\n",
+    "    'unit': 'USD/GWa',\n",
```

Comment:
Please make sure the unit is correct (USD/GWa seems should be USD/kWa).

Reply:
done